fineract-provider/src/main/java/org/apache/fineract/cob/loan/LoanAccountsStayedLockedBusinessEvent.java

```diff
@@ -39,4 +39,9 @@ public String getType() {
     public String getCategory() {
         return CATEGORY;
     }
+
+    @Override
```

Comment:
Null?

Reply:
In case a single aggregate root cannot be identified, for example the LoanAccountsStayedLockedBusinessEvent. Multiple accounts stayed locked so there's no singular event hence it doesn't matter which producer picks up the message to be sent.